codecov.yml

```diff
@@ -1,3 +1,4 @@
 coverage:
   ignore:
     - "Example"
+    - "FetchRequests/Sources/SwiftUI"
```

Comment:
Since so much of SwiftUI concerns a lifecycle we do not control, any test cases would be _very_ truthy.